src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -52,10 +54,19 @@
 import static org.opensearch.security.dlic.rest.support.Utils.hash;
 
 public class InternalUsersApiAction extends PatchableResourceApiAction {
```

Comment:
An extensive refactor [1] to how classes are declared and default behaviors was merged recently, you'll need to resolve these conflicts before this can be reviewed.

- [1] https://github.com/opensearch-project/security/pull/3123

Reply:
Unresolving it, taking longer than I expected. Having difficulties as tests are timing out for some reason

Reply:
While you are working on the tests could you push your latest changes so they can be looked at while you troubleshoot?